src/shared/components/navbar.tsx

```diff
@@ -5,48 +5,60 @@ import { i18n, LANGUAGES } from "../i18next";
 import classNames from "classnames";
 import { SELECT_CLASSES } from "./common";
 
-const NavLink = ({ content }) => <li className="text-gray-400">{content}</li>;
+const NavLink = ({ content }) => (
+  <li className="text-gray-400 w-full md:w-fit">{content}</li>
+);
+
+const LinkClass = "text-xl px-4  py-2 md:text-lg";
```

Comment:
Small nitpick, but I'd use either camelCase or SCREAMING_SNAKE_CASE for this constant name. Usually when I see PascalCase used in JS/TS, it's either for a class or a UI component.

Reply:
Fixed